content/en/docs/reference/command-line-tools-reference/feature-gates/kubelet-pod-resources-dynamic-resources.md

```diff
@@ -11,7 +11,7 @@ stages:
     fromVersion: "1.27"  
 ---
 Extend the kubelet's pod resources gRPC endpoint to
-to include resources allocated in `ResourceClaims` via `DynamicResourceAllocation` API.
-See [resource allocation reporting](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources) for more details.
-with information about the allocatable resources, enabling clients to properly
+include resources allocated in `ResourceClaims` via `DynamicResourceAllocation` API.
+See [resource allocation reporting](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources)
+for more details. With information about the allocatable resources, enabling clients to properly
```

Comment:
This sentence doesn't make sense. I understand that you just removed the period in between, but even before the removal, it didn't make sense.

Reply:
Changed the link text to the exact section title. Does it make sense a little more? I have no more ideas...